### ISSUE TITLE
Mention "shellcheck" in a comment without it being parsed as a directive?

### ISSUE THREAD

**Issue by roi3ert** [koalaman/shellcheck#2408]
#### For bugs
- SC1072, SC1073
- My shellcheck version (`shellcheck --version` or "online"): 
```sh
$ shellcheck --version
ShellCheck - shell script analysis tool`
version: 0.7.2
license: GNU General Public License, version 3
website: https://www.shellcheck.net
```

- [x] The rule wiki pages do not already cover this (https://shellcheck.net/wiki/SC1072, https://shellcheck.net/wiki/SC1073)
- [x] I tried on https://www.shellcheck.net/ and verified that this is still a problem on the latest commit

#### For new checks and feature suggestions
- [x] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

I have a comment line (line 5) that mentions "shellcheck" as the first word of the comment line:

```sh

#!/bin/bash
# The purpose of this script is blah blah blah...
#
# To check this script for issues, use shellcheck as follows:
#    shellcheck this_script.sh
#
# And then...
#

echo "foo bar baz"
```

#### Here's what shellcheck currently says:

```sh
$ shellcheck this_script.sh 

In this_script.sh line 5:
#    shellcheck this_script.sh
^-- SC1073: Couldn't parse this shellcheck directive. Fix to allow more checks.
                    ^-- SC1072: Expected '=' after directive key. Fix any mentioned problems and try again.

For more information:
  https://www.shellcheck.net/wiki/SC1072 -- Expected '=' after directive key....
  https://www.shellcheck.net/wiki/SC1073 -- Couldn't parse this shellcheck di...
```

Is there some way to get this to work?  I know I could combine lines 4 and 5 and it works:

```sh

#!/bin/bash
# The purpose of this script is blah blah blah...
#
# To check this script for issues, use shellcheck as follows:  shellcheck this_script.sh
#
# And then...
#

echo "foo bar baz"
```

Are there any better options?

**Comment by TinCanTech** [comment]
Possibly: `##    shellcheck this_script.sh` ?

**Comment by roi3ert** [comment]
> Possibly:
> 
> ```
> ##    shellcheck this_script.sh
> 
> #   > shellcheck this_script.sh
> ```
> 
> ?

Your second solution reminded me of using a backslash:
https://tldp.org/LDP/GNU-Linux-Tools-Summary/html/c1195.htm
```
#!/bin/bash
# The purpose of this script is blah blah blah...
#
# To check this script for issues, use shellcheck as follows:
#    \shellcheck this_script.sh
#
# And then...
#

echo "foo bar baz"
```
It can be copy/pasted to the command line:

`$ \shellcheck this_script.sh`

I'm leaning towards either that or your first solution (double pound sign).
